### ISSUE TITLE
NTR: Cystathionine beta-synthase enzyme activity low in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9070]
New term request Cystathionine beta-synthase enzyme activity low in fibroblasts (segal_040116143023):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000357
- label: Reduced cystathionine beta-synthase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:5647845
- def: Acitivity of cystathionine beta-synthase (EC 4.2.1.22) below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9070
- synonyms:
